src/metric-engine/src/metadata_region.rs

```diff
@@ -65,16 +61,23 @@ impl MetadataRegion {
     ///
     /// This method will check if the table key already exists, if so, it will return
     /// a [TableAlreadyExistsSnafu] error.
-    pub async fn add_table(&self, region_id: RegionId, table_id: TableId) -> Result<()> {
-        let region_id = utils::to_metadata_region_id(region_id);
-        let table_key = Self::concat_table_key(table_id);
+    pub async fn add_logical_region(
+        &self,
+        physical_region_id: RegionId,
+        logical_region_id: RegionId,
+    ) -> Result<()> {
+        let region_id = utils::to_metadata_region_id(physical_region_id);
+        let region_key = Self::concat_region_key(logical_region_id);
 
         let put_success = self
-            .put_conditionally(region_id, table_key, String::new())
+            .put_conditionally(region_id, region_key, String::new())
```

Comment:
Question: It seems there are no conditions in the parameters, why not use `put_if_absent` or `put_if_not_exists`?